.github/workflows/ci.yml

```diff
@@ -73,6 +73,44 @@ jobs:
       - name: Run integration tests
         if: ${{ env.B2_TEST_APPLICATION_KEY != '' && env.B2_TEST_APPLICATION_KEY_ID != '' }}
         run: nox -vs integration
+  cli-test:
+    needs: lint
+    env:
+      B2_TEST_APPLICATION_KEY: ${{ secrets.B2_TEST_APPLICATION_KEY }}
+      B2_TEST_APPLICATION_KEY_ID: ${{ secrets.B2_TEST_APPLICATION_KEY_ID }}
+      INSTALL_SDK_FROM: ../b2-sdk-python
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        python-version: [ 3.5, 3.9, pypy3 ]
```

Comment:
Why these versions?

Reply:
The lowest, the highest and pypy. So we save time by not running tests on 3.6, 3.7 and 3.8.